go/netutil/netutil_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"testing"
 )
 
-func checkDistribution(t *testing.T, data []*net.SRV, margin float64) {
+func checkDistribution(t *testing.T, rand *rand.Rand, data []*net.SRV, margin float64) {
```

Comment:
Given that the new `golang` behavior is to auto-seed the random generator on startup, why would we want to pass a `rand * Rand` argument to these functions? Wouldn't we just want them to generate random values as opposed to deterministic ones?

Reply:
there were a few test cases that relied on `rand.Seed(1)` before the start of the test, so we update this function to take a rand instance which is either seeded to 1 or to time.Now() (aka "random value")

Reply:
I'm just hesitant to require this additional argument in the function signature. Is it possible to modify the tests to not rely on a reproducible ordering?

Reply:
I hear you, but I am hesitant to change it, and my brief read (without fully digging in/understanding, so admitting that up front) seems to be that it is definitionally dependent on randomness: https://github.com/vitessio/vitess/pull/42/files#diff-b0fbcaba3872685e058eab7c65e0397e27dab0b0df92694e46fbcb67be911f28

Reply:
Yeah, for get it, I notice that these functions are only called internally in the same file (and in the unit test file), so not worth bothering.